Source/AwsCommonRuntimeKit/Utilities.swift

```diff
@@ -160,3 +160,14 @@ func withByteCursorFromStrings<R>(
         }
     }
 }
+
+func withOptionalUnsafePointer<T, R>(
+        _ arg1: T?, _ body: (UnsafePointer<T>?) -> R
```

Comment:
I could totally see you adding more overloads of this function in the future that take `<T1, T2, R>`, `<T1, T2, T3, R>`, etc

Reply:
Nuked the function :joy: